packages/base/src/Keys.js

```diff
@@ -131,6 +131,10 @@ const isUpShift = event => (event.key ? (event.key === "ArrowUp" || event.key ==
 
 const isDownShift = event => (event.key ? (event.key === "ArrowDown" || event.key === "Down") : event.keyCode === KeyCodes.ARROW_DOWN) && checkModifierKeys(event, false, false, true);
 
+const isLeftShift = event => (event.key ? (event.key === "ArrowLeft" || event.key === "Left") : event.keyCode === KeyCodes.ARROW_LEFT) && checkModifierKeys(event, false, false, true);
```

Comment:
This might also sound like this function tests for whether the left shift key was pressed, but i decided to go with it to be consistent with the naming `is[Key][Modifier]`.

Reply:
I also think it's confusing, but it's consistent